src/function.rs

```diff
@@ -372,6 +375,30 @@ fn path_exists(evaluator: &Evaluator, path: &str) -> Result<String, String> {
   )
 }
 
+fn pick(_evaluator: &Evaluator, count: &str, alphabet: Option<&str>) -> Result<String, String> {
+  let alphabet = alphabet.unwrap_or("0123456789abcdef");
```

Comment:
let's move this out to a `const` at the top -- the default alphabet should probably be called out (and possibly reusable) somewhere.